File: XOutput.Api/Websocket/Input/InputDeviceInputResponse.cs
using System.Collections.Generic;

namespace XOutput.Websocket.Input
{
    public class InputDeviceInputResponse : MessageBase
    {
        public const string MessageType = "InputDeviceInputFeedback";
        public List<InputDeviceSourceValue> Sources { get; set; }
        public List<InputDeviceTargetValue> Targets { get; set; }
        public InputDeviceInputResponse()
        {
            Type = MessageType;
        }
    }
}

File: Web/src/api/websocket/input/InputDeviceInputResponse.ts
import { MessageBase } from "../MessageBase";
import { InputDeviceSourceValue } from "./InputDeviceSourceValue";
import { InputDeviceTargetValue } from "./InputDeviceTargetValue";

export const InputDeviceInputResponseType = "InputDeviceInputFeedback";

export type InputDeviceInputResponse = MessageBase<typeof InputDeviceInputResponseType> & 
{
    sources: InputDeviceSourceValue[];
    targets: InputDeviceTargetValue[];
};
